### ISSUE TITLE
[Documentation Request] Confusion in Reading Reports

### ISSUE THREAD

**Issue by ericaVoss** [OHDSI/AchillesWeb#51]
I'm writing to request more documentation in ACHILLES as well as better report labeling within the application.  

I had this experience today which highlights the need for it:

>*Today I was reviewing a condition via one SNOMED code.  I looked the one SNOMED code up in the CONDITIONS report in ACHILLES and saw that, let's say, it had an n=1000 patients with the conditions.  I glanced at the report and noticed the Age Decile report looked interesting.*

>*I wanted to explore this further, so I defined a cohort in CIRCE (which had my same n=1000 cohort) and tried to use ATLAS/HERACLES to do additional analysis.  On the Cohort Specific Reports I noticed that the Age Decile report looked very different than the one I had in ACHILLES.  I was worried there was an issue with one of the reports.*

>*Patrick was able to identify the issue was they were actually different reports:*
> - *ACHILLES:  graph is showing you the proportion of patients with the condition amongst the overall database population, stratified by age/gender/year*
> - *ATLAS/HERACLES:  graph is showing you the proportion of patients with the condition recorded in each age/gender/year strata amongst the cohort of patients who have serum sickness who have observation period in that year.*

Without Patrick, I don't think I would have been able to understand the difference based on the text in the applications alone. 

Taking this bit broader, how are we going to support documentation going forward?  Is there a process we could have that would allow multiple people to participate.  Maybe it is putting it in the WIKI or maybe putting it in a DB table that can be easily updated.  I don't have the answer, but I think documentation is a issue in general.

This is related to:
https://github.com/OHDSI/Heracles/issues/20

**Comment by cgreich** [comment]
@ericaVoss:

I would write make it easy on us: Just link this Forum discussion to a little icon we need to add to Achilles next to each report. That way, the community can discuss every report, if it's worth it, without us having to create a bureaucracy. 

However, if you want to start a wiki page to be linked to an Achilles page or report just open any Wiki page, change the name of the page in the URL and viola, you got an empty Wiki you can start editing.

And then you have to edit the Achilles page to link stuff up.

**Comment by ericaVoss** [comment]
@cgreich yes I agree it is more realistic if the community can participate.  I think the Wiki option sounds good.

**Comment by cgreich** [comment]
You can't discuss on Wiki. Only on the Forum. So, still text - Wiki, discussion - Forum.
